src/CabanaPD_Boundary.hpp

```diff
@@ -21,8 +21,20 @@
 namespace CabanaPD
 {
 
+struct Rectilinear
+{
+};
+struct Cylinder
+{
+};
+
+// Forward declaration.
+template <class GeometryType>
+struct RegionBoundary;
+
 // Define a plane or other rectilinear subset of the system as the boundary.
```

Comment:
This is not strictly speaking a plane. Also, what does it mean "or other rectilinear subset"? It seems this is specific to a rectangular prism.

Reply:
Removed from this PR